packages/api/src/platforms/vtex/utils/cookies.ts

```diff
@@ -122,6 +123,25 @@ export const getWithAutCookie = (ctx: ContextForCookies) => {
   }
 }
 
+export const getWithAppKeyAndToken = () => {
```

Comment:
do you think we can move it to another file? Something more related to auth? 🤔

Reply:
Makes sense to me. done [here](https://github.com/vtex/faststore/pull/3055/commits/da32747c6c0a48b74ad390d02addbfc9383cd953)